Decred Wallet/Custom Views/AccountTableView.swift

```diff
@@ -0,0 +1,168 @@
+        let defaultAccount = acc?.filter({ $0.isDefault}).first
+        if (defaultAccount != nil) {
+            let defaultAcc: [DcrlibwalletAccount] = [defaultAccount!]
+            self.listData?.append(defaultAcc)
+            
+            let walletAccountsArr: [DcrlibwalletAccount] = (acc?.filter({!$0.isHidden && $0.number != INT_MAX }))!
+            self.listData?.append(walletAccountsArr)
+        }else{
+           let walletAccountsArr: [DcrlibwalletAccount] = (acc?.filter({!$0.isHidden && $0.number != INT_MAX }))!
+           self.listData?.append(walletAccountsArr)
+        }
+        
+        self.tableView?.reloadData()
+    }
+    
+    //MARK: tableViewDelegate
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return self.listData!.count
+    }
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        let walletAccountsArr: [DcrlibwalletAccount]? = self.listData![section]
+        
+        return walletAccountsArr!.count
+    }
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 78
+    }
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 30
+    }
+    func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        return 0.1
+    }
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell:AccountTableViewCell = tableView.dequeueReusableCell(withIdentifier: "AccountTableViewCell", for: indexPath) as! AccountTableViewCell
+        
+        let cellBgView: UIView? = UIView.init()
+        cellBgView?.backgroundColor = .white
+        cell.selectedBackgroundView = cellBgView
+        
+        let walletAccountsArr: [DcrlibwalletAccount] = (self.listData?[indexPath.section])!
+        let  walletAccount = walletAccountsArr[indexPath.row]
+        
+        cell.setAccount(walletAccount)
+        
+        return cell
+    }
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let walletAccountsArr: [DcrlibwalletAccount] = (self.listData?[indexPath.section])!
+        let  walletAccount = walletAccountsArr[indexPath.row]
+
+        let walletName = indexPath.section==0 ? "Default":"Wallets"
+        
+        self.selectedAccount!(walletAccount,walletName)
+        
+        self.hideView()
+    }
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let header = UIView.init(frame: CGRect(x: 0, y: 0, width: 0, height: 30))
+        header.backgroundColor = .white
+        
+        let lab = UILabel.init(frame: CGRect(x: 16, y: 14, width: tableView.frame.width-30, height: 16))
+        lab.font = .systemFont(ofSize: 14)
+        lab.textColor = .init(red: 0.24, green: 0.35, blue: 0.45, alpha: 1)
+        lab.text = section==0 ? "Deafult":"Wallets"
```

Comment:
```suggestion
        lab.text = section==0 ? "Default":"Wallets"
```
Btw: these strings should come from `LocalizedStrings`